src/main/java/de/tum/in/www1/artemis/repository/CourseRepository.java

```diff
@@ -69,4 +70,12 @@
     List<Course> findAllByShortName(String shortName);
 
     Optional<Course> findById(long courseId);
+
+    @Query("""
+            select s.submissionDate as day, u.login as username
+            from User u, Submission s, StudentParticipation p
+            where s.participation.exercise.course.id = :#{#courseId} and s.participation.id = p.id and p.student.id = u.id and s.submissionDate >= :#{#startDate} and s.submissionDate <= :#{#endDate} and u.login not like '%test%'
```

Comment:
https://github.com/ls1intum/Artemis/blob/243eb3a86411bdef1df7adb0e95951d31a5d7014/src/main/java/de/tum/in/www1/artemis/repository/ExerciseRepository.java#L78-L80
explicitly excludes tutors etc. for the student score. We should think about either doing this here as well, or later on change the statement I linked to be consistent.

Reply:
That's a reasonable remark, i can remove instructors and tutors from the query.